### PR TITLE
[MIG][10.0-Issue175-T23293] hr_timesheet_auto_creation.

### PR DESCRIPTION
**Pull request by sudhir-serpentcs**
Migrated hr_timesheet_auto_creation in v10.
@elicoidal @seb-elico @victormartinelicocorp Please review.
This PR refers to issue #175

### CODE REVIEW

File: hr_timesheet_auto_creation/README.rst [hunk 1]
```diff
@@ -54,20 +56,10 @@ Maintainer
...
 This module is maintained by Elico Corporation.
 
-Elico Corp is an innovative actor in China, Hong-Kong and Singapore servicing
-well known international companies and as well as local mid-sized businesses.
-Since 2010, our seasoned Sino-European consultants have been providing full
-range Odoo services:
+Elico Corporation offers consulting services to implement open source management
+software in SMEs, with a strong involvement in quality of service.
 
-* Business consultancy for Gap analysis, BPM, operational work-flows review.
```
Comment (elicoidal) [inline hr_timesheet_auto_creation/README.rst]: please use latest template

**Review by elicoidal — CHANGES_REQUESTED** [review]
details. LGTM pending tests

File: hr_timesheet_auto_creation/README.rst (hunk 1 above)
Comment (sudhir-serpentcs) [reply hr_timesheet_auto_creation/README.rst]: @elicoidal I already have used latest template.
Correct me if I am wrong and send me a new template or it's content.

**Review by elicoidal — APPROVED** [review]
LGTM

**Review by victor-ditay — APPROVED** [review]
To be tested

**Pull request merged**